middleware/middleware-1.3.39.md

```diff
@@ -0,0 +1,42 @@
+---
+slug: /release-notes/middleware/1.3.39
+title: Version 1.3.39
+---
+
+# fiskaltrust.Middleware 1.3.39 (Germany)
+_September 05, 2022_
+
+In this version of the Middleware,we made it possible to run the German middleware on an ARM device, we've also fixed a bug concerning Fail-transactions not removing the failed start transactions on the FailedStartTransaction table in the database.
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Feature: ARM support for the German middleware
+We now offer support for customers using ARM devices and wish to implement our German middleware. These are the configuration working on ARM devices :
+- SQLITE
```

Comment:
```suggestion
- SQLite
```

Reply:
IMO not completely correct, sorry 😅 fiskaly and Cryptovision should also work, as well as the other Queues.

Maybe it would be nicer to highlight that _most_ things are working, except for some infrequently used TSEs, and TSEs that don't have support for Arm in their SDK. These don't work IMO:
- Epson SCU
- Diebold Nixdorf SCU
- Swissbit Hardware SCU on Arm64 (32-bit works, there's no SDK für Arm64, unfortuantely)

The remaining packages should be good :)